operator-wg/whitepaper/05_Frameworks.md

```diff
@@ -3,9 +3,25 @@
 ## Operator Frameworks for Kubernetes
 **(Current) Issue: https://github.com/cncf/sig-app-delivery/issues/40**
 
-[Framework_1](051_Framework_1.md) | Not even created | |
+## [Framework_1](051_Framework_1.md)
+
+- Add Short description of the framework
+- How does it work (in a Nutshell)?
+- What makes it unique?
```

Comment:
This question about uniqueness seems ambiguous and highly opinionated. Maybe it is better to ask contributors to describe the delivered value or something else more specific?

Reply:
Applied a change based on your suggestion. Please take a look. Thanks!